src/com/xilinx/rapidwright/util/RunVivado.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (c) 2021-2022, Xilinx, Inc.
+ * Copyright (c) 2022-2023, Advanced Micro Devices, Inc.
+ * All rights reserved.
+ *
+ * Author: Zak Nafziger, Xilinx Research Labs.
+ *
+ * This file is part of RapidWright.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package com.xilinx.rapidwright.util;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Utility methods to run a tcl script through vivado and parse the stdout
+ * @author zakn
+ *
+ */
+public class RunVivado {
```

Comment:
Maybe call this class `Vivado` to reflect that it (will) contain all sorts of Vivado utilities.

Reply:
Calling this class just `Vivado` seems wrong since it is an interface/collection of utilities to access vivado. Maybe `VivadoUtils`?

Reply:
Maybe `VivadoTools` then?